src/main/resources/templates/pdf.css

```diff
@@ -107,7 +107,6 @@ body {
 #qr-code {
     width: 300px;
     height: 300px;
-    margin-left: -20px;
```

Comment:
This change makes the QR-Code being indented. Is this change just by mistake or does it have to look like that?

Reply:
For some reason the QR code is smaller when gerated localy. Therefore I removed the margin to make it look good on the other environments.